h2o-docs/src/product/data-science/gbm-faq/generated_metrics.rst

```diff
@@ -9,8 +9,10 @@ What evaluation metrics are available for GBM?
 - MAE
 - RMSLE
 - Mean Residual Deviance
+- Mean Residual Deviance Not Optimized
+
+**Notes**: MSE = Mean Residual Deviance for Gaussian distributions. Also, look at Mean Residual Deviance when using quantile distributions. Mean Residual Deviance Not Optimized is currently only implemented for Poisson distribution. Mean Residual Deviance Not Optimized is calculated from the original deviance formula. Mean Residual Deviance uses an optimized deviance formula for some distributions which provides a different output than the original deviance formula would. 
```

Comment:
@syzonyuliia Can you provide the formulae for Hannah for both the original Mean Residual deviance and the mean residual deviance not optimized here?

Reply:
@wendycwong I'm not sure if this is appropriate here, as there are several distributions and each has its own formula

Reply:
@wendycwong @syzonyuliia Neither Mean Residual Deviance nor Mean Residual Deviance Not Optimized are on the [performance and prediction page for evaluation metrics](https://docs.h2o.ai/h2o/latest-stable/h2o-docs/performance-and-prediction.html#evaluation-model-metrics). Perhaps we could add two small sections for each and include their formulae there?

Reply:
Sorry @hannah-tillman :  Please close this PR. There is only one deviance calculation, not two.